Sources/JExtractSwiftLib/JNI/JNISwift2JavaGenerator+JavaBindingsPrinting.swift

```diff
@@ -255,13 +266,18 @@ extension JNISwift2JavaGenerator {
     if decl.swiftNominal.isSendable {
       printer.print("@ThreadSafe // Sendable")
     }
+    var modifiers = ["public"]
+    if decl.parent != nil {
+      modifiers.append("static")
+    }
+    modifiers.append(contentsOf: ["final", "class"])
```

Comment:
Leave `class` out of that one, not a modifier -- just normal as it was and prefix with modifiers